ztp/source-crs/PerformanceProfile.yaml

```diff
@@ -30,3 +30,7 @@ spec:
   # To use the standard (non-realtime) kernel, set enabled to false
   realTimeKernel:
     enabled: true
+  workloadHints:
```

Comment:
Can you add a comment here to describe what this section is for. Does not need to be comprehensive documentation but a minimal indication of what workloadHints are for and what behavior these default values result in.

Reply:
We should add a link to the official documentation vs re-documenting.